File: PHP/Baju.php
<?php
require_once "Aksesoris.php";

// kelas turunan dari Aksesoris: Baju
class Baju extends Aksesoris {
    private $untuk;
    private $size;
    private $merk;

    // konstruktor
    public function __construct($id, $nama_produk, $harga, $stok, $foto_produk, $jenis, $bahan, $warna, $untuk, $size, $merk) {
        parent::__construct($id, $nama_produk, $harga, $stok, $foto_produk, $jenis, $bahan, $warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    // getter
    public function getUntuk() { return $this->untuk; }
    public function getSize() { return $this->size; }
    public function getMerk() { return $this->merk; }

    // menampilkan data dalam format tabel HTML
    public function tampilkan() {
        echo "<tr>
                <td>{$this->id}</td>
                <td>{$this->nama_produk}</td>
                <td>Rp" . number_format($this->harga, 0, ',', '.') . "</td>
                <td>{$this->stok}</td>
                <td><img src='{$this->foto_produk}' width='50' height='50'></td>
                <td>{$this->jenis}</td>
                <td>{$this->bahan}</td>
                <td>{$this->warna}</td>
                <td>{$this->untuk}</td>
                <td>{$this->size}</td>
                <td>{$this->merk}</td>
              </tr>";
    }
}
?>

File: Python/Baju.py
from Aksesoris import Aksesoris

# kelas turunan dari Aksesoris: Baju
class Baju(Aksesoris):
    def __init__(self, id, nama_produk, harga, stok, jenis, bahan, warna, untuk, size, merk):
        super().__init__(id, nama_produk, harga, stok, jenis, bahan, warna)
        self.untuk = untuk
        self.size = size
        self.merk = merk

    # getter
    def get_untuk(self):
        return self.untuk

    def get_size(self):
        return self.size

    def get_merk(self):
        return self.merk

    # menampilkan data dalam format tabel
    def tampilkan(self):
        print(f"{self.id:<5} {self.nama_produk:<15} {self.harga:<10} {self.stok:<5} {self.jenis:<10} {self.bahan:<10} {self.warna:<10} {self.untuk:<10} {self.size:<5} {self.merk:<10}")